backend/src/monarch_py/utils/utils.py

```diff
@@ -58,10 +58,9 @@ def dict_diff(d1, d2):
 
 def set_log_level(log_level: str):
     """Sets the log level for the application."""
-    import loguru
+    from loguru import logger
 
-    loguru.logger.remove()
-    loguru.logger.add(sys.stderr, level=log_level)
+    logger.configure(handlers=[{"sink": sys.stdout, "level": "ERROR"}])
```

Comment:
It looks like this ends ignoring the log level that's passed in?

Reply:
it totally was yeah, my bad